packages/utils/src/dal/mikro-orm/mikro-orm-repository.ts

```diff
@@ -716,8 +716,12 @@ export function mikroOrmBaseRepositoryFactory<T extends object = object>(
         managed: false,
         persist: false,
       })
+      const loadedData: object = {}
+      for (const property of (created as any).__helper.__loadedProperties) {
+        loadedData[property] = created[property]
+      }
```

Comment:
nit: we could return the loaded data directly.

```suggestion
      const loadedData: object = { id: (created as any).id }
      
      for (const property of (created as any).__helper.__loadedProperties) {
        loadedData[property] = created[property]
      }
```

Reply:
@riqwan found another solution, and this fixes the issue with non-persist columns not being removed (and resulted in a bit of cleanup).